[CI] GenerativeIT test failing

**Build Scans:**
- [elasticsearch-intake #30082 / part3](https://gradle-enterprise.elastic.co/s/he3r3b3sz42hs)
- [elasticsearch-pull-request #98900 / part-3-fips](https://gradle-enterprise.elastic.co/s/slrn6ftf32lti)
- [elasticsearch-pull-request #98683 / part-3](https://gradle-enterprise.elastic.co/s/xfju7pmprqv3c)
- [elasticsearch-pull-request #98648 / part-3](https://gradle-enterprise.elastic.co/s/r3dnzx3cbuabs)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT.test" -Dtests.seed=E0EE236DBF5CA2A0 -Dtests.locale=kok-Deva -Dtests.timezone=Atlantic/Canary -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: query: from multivalue_geom*,no_mapping_sample_data | rename `id` AS `NfHBAZaCUm`, contains AS intersects | drop `intersects` | stats  uuHzOSMmAGp = count(NfHBAZaCUm), `shape` = median(NfHBAZaCUm) + max(NfHBAZaCUm) + max(NfHBAZaCUm), `smaller` = count(NfHBAZaCUm) + count(NfHBAZaCUm), xxGKVoyhS = count_distinct(NfHBAZaCUm), kuNvvqRtEgq = count(*)
exception: method [POST], host [http://[::1]:34943], URI [/_query?pretty=true&error_trace=true], status line [HTTP/1.1 500 Internal Server Error]
Warnings: [No limit defined, adding default limit of [1000]]
¿eerror¿jroot_cause¿dtypetclass_cast_exceptionfreasonyclass org.elasticsearch.compute.data.LongVectorBlock cannot be cast to class org.elasticsearch.compute.data.BytesRefBlock (org.elasticsearch.compute.data.LongVectorBlock and org.elasticsearch.compute.data.BytesRefBlock are in unnamed module of loader java.net.URLClassLoader @1f9d4b0e)kstack_traceyorg.elasticsearch.ElasticsearchException$1: class org.elastic
[truncated]
```

**Issue Reasons:**
- [main] 4 failures in test test (1.5% fail rate in 268 executions)
- [main] 2 failures in step part-3 (1.6% fail rate in 124 executions)
- [main] 3 failures in pipeline elasticsearch-pull-request (2.4% fail rate in 124 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -657,12 +657,6 @@ tests:
 - class: org.elasticsearch.index.codec.vectors.es93.ES93HnswBinaryQuantizedBFloat16VectorsFormatTests
   method: testFloatVectorScorerIteration
   issue: https://github.com/elastic/elasticsearch/issues/136566
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/136584
-- class: org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/136584
 
 # Examples:
 #

--- a/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
+++ b/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
@@ -68,6 +68,10 @@ public abstract class GenerativeRestTest extends ESRestTestCase implements Query
         "optimized incorrectly due to missing references", // https://github.com/elastic/elasticsearch/issues/131509
         "long overflow", // https://github.com/elastic/elasticsearch/issues/135759
         "can't build page out of released blocks", // https://github.com/elastic/elasticsearch/issues/135679
+        "cannot be cast to class", // https://github.com/elastic/elasticsearch/issues/133992
+        "can't find input for", // https://github.com/elastic/elasticsearch/issues/136596
+        "unexpected byte", // https://github.com/elastic/elasticsearch/issues/136598
+        "Rule execution limit", // https://github.com/elastic/elasticsearch/issues/136599
 
         // Awaiting fixes for correctness
         "Expecting at most \\[.*\\] columns, got \\[.*\\]", // https://github.com/elastic/elasticsearch/issues/129561

